packages/react/src/components/ErrorSummary/ErrorSummary.mdx

```diff
@@ -24,19 +27,81 @@ import { ErrorSummary } from '@digdir/designsystemet-react';
+## Eksempler
+
+### Bruk med tekstfelt
 
 <Canvas of={ErrorSummaryStories.WithForm} />
 
-## aria-role='alert'
+### aria-role='alert'
 
-`ErrorSummary` har `aria-role='alert'` som standard. Dette betyr at skjermlesere vil lese opp innholdet i `ErrorSummary` automatisk.
+`ErrorSummary` har `aria-role='alert'` som standard. Dette betyr at skjermlesere vil lese opp innholdet i `ErrorSummary` automatisk. 
+Det er fortsatt viktig at `ErrorSummary` får fokus ved submit slik at brukeren får med seg at det er feil eller mangler i skjemaet.
 
 <Canvas of={ErrorSummaryStories.ShowHide} />
 
+## Retningslinjer
+
+`ErrorSummary` brukes til å vise en oppsummering når det er feil eller mangler i noe brukeren har gjort. En oppsummering kan gjelde for én eller flere feil.
+
+**Pass på at**
+- brukerne kan gå direkte til feilene de må rette opp, og fokuset blir flyttet dit
+- overskriften “For å gå videre må du rette opp følgende feil” brukes
+- lenkene i oppsummeringen stemmer med feilmeldingene de lenker til
+
+<br/>
+
+ 
+
+
+
+<br />
+
+### Plassering
+
+Vi anbefaler å vise `ErrorSummary` like over Neste/Send inn-knappen. Dette for 
+at brukerne skal forstå sammenhengen mellom feilen og hvorfor de blir hindret i å gå videre. 
+
+I noen tilfeller kan det likevel være bedre å vise oppsummeringen i toppen, det kan du lese mer om i [artikkelen om feilmeldinger](https://designsystemet.no/monstre/feilmeldinger#oppsummering-av-flere-feilmeldinger).
+
+`ErrorSummary` skal inneholde alle feilmeldingene som er på siden, så brukerne kan navigere direkte 
+til feilene og manglene.
+
+
+Det skal være lenker fra `ErrorSummary` til der feilen er på siden. Når brukeren trykker på en av lenkene, 
+skal fokus flyttes til der feilen er.
+
+Lenketeksten i `ErrorSummary` må være identisk med feilen den lenker til.
+
+<br/>
+
+### Synlighet
+
+Oppsummeringen skal ikke være synlig uten at brukeren foretar seg noe. Den skal bli synlig først når brukeren har utført en handling om utløser feil.
+
+- Hvis vi validerer fortløpende (feks. ved `onBlur` på hvert felt), venter vi med å vise ErrorSummary til brukeren prøver å sende inn skjemaet.
+- Det er viktig at komponenten får fokus når den blir vist, slik at brukeren får med seg at det er feil eller mangler i skjemaet.
+- Feilmeldingene i oppsummeringen skal være formulert lik feilmeldingen ved feltet.
+- Feilmeldingene skal lenke direkte til feltet det gjelder.
+- Hvis feilen gjelder flere felt, lenker vi til det første tilfellet av feilen.
+
+<br/>
+
+## Tilgjengelighet
+
+`ErrorSummary` har `aria-role='alert'` som standard. Dette betyr at skjermlesere vil lese opp innholdet i `ErrorSummary` automatisk når den vises.
+Alle punkt i listen er lenker, og oppfører seg som en lenke.
+
+
+** Pass på at**
+- Når oppsummeringen blir synlig, flyttes fokus dit
+- Når brukeren trykker på en lenke, flyttes fokus til stedet hvor feilen eller mangelen oppstod
+
+<br />
+
 ### `ErrorSummary.Item`
 
 <ArgTypes of={ErrorSummary.Item} />
```

Comment:
Maybe move this up top close to ErrorSummary preview? Seems weird with it just dangling at the very end

Reply:
I think we need to look at this across our documentation later, other components also have this at the bottom